### PR TITLE
Responsive home

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

**Review by julianhahn — CHANGES_REQUESTED** [review]
on my screen it looks like this:
image

Could you test your screen again, before I change the code and it breaks the view on your screen?

File: packages/app/src/views/Home.vue [hunk 1]
```diff
@@ -1,34 +1,49 @@
 <template>
-  <div class="home mx-auto max-w-lg h-full w-full flex flex-col">
-    <header class="h-1/5 flex items-end justify-center text-4xl text-gray-800">
+  <div class="home flex flex-col">
```
Comment (julianhahn) [inline packages/app/src/views/Home.vue:2]:
```suggestion
  <div class="home w-full flex flex-col">
```
I think there is just a w-full missing

File: packages/app/src/App.vue [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <template>
-  <div class="app flex">
-    <router-view />
+  <div class="app bg-green-800 flex">
```
Comment (julianhahn) [inline packages/app/src/App.vue:2]: supa nice background 🎉

File: packages/app/src/views/Home.vue [hunk 3]
```diff
@@ -1,34 +1,49 @@
...
             />
           </div>
         </GroupDropDownItem>
       </GroupDropDown>
     </main>
-    <footer class="flex-grow flex items-center justify-center text-4xl text-gray-800"></footer>
+
+    <footer class="flex text-gray-800 py-5">
+      <span class="text-xl text-center w-full"
+        >&copy; {{ new Date().getFullYear() }} - Made with ❤️ by
```
Comment (julianhahn) [inline packages/app/src/views/Home.vue:36]: i like this, so much

**Review by julianhahn — CHANGES_REQUESTED** [review]

File: packages/app/src/views/Home.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/views/Home.vue:2]: Stange I do not have this error on any of my responsive stages, but adding `w-full` does not break it.

**Review by JanEberlein — APPROVED** [review]
looks good

File: packages/app/src/App.vue (hunk 2 above)
Comment (tcoro) [inline packages/app/src/App.vue:2]: I would suggest another color as background based on
https://color.adobe.com/de/create/color-wheel

maybe use ```bg-indigo-800``` ```bg-blue-900``` or ```bg-blue-800```

**Review by tcoro — COMMENTED** [review]
I suggest changing the background color

**Review by julianhahn — APPROVED** [review]

File: packages/app/src/App.vue (hunk 2 above)
Comment (anbraten) [reply packages/app/src/App.vue:2]: I checked your suggestions, but currently I would stick to the color we have.

**Pull request merged**